sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -66,15 +68,32 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential) : this(endpoint
         /// <param name="endpoint">The service endpoint to use.</param>
         /// <param name="credential">The <see cref="TokenCredential"/> instance to use for authentication.</param>
         /// <param name="options">The <see cref="LogsQueryClientOptions"/> instance to use as client configuration.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public LogsQueryClient(Uri endpoint, TokenCredential credential, LogsQueryClientOptions options)
         {
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(endpoint, nameof(endpoint));
 
             Endpoint = endpoint;
             options ??= new LogsQueryClientOptions();
-            var authorizationScope = $"{(string.IsNullOrEmpty(options.Audience?.ToString()) ? LogsQueryAudience.AzurePublicCloud : options.Audience)}";
-            authorizationScope += "//.default";
+
+            // Set authorization scope from Endpoint if Audience is not set.
+            string authorizationScope = "";
+            if (string.IsNullOrEmpty(options.Audience?.ToString()))
+            {
+                options.Audience = endpoint.AbsoluteUri;
```

Comment:
Is this still needed?

Reply:
We want the Audience and Endpoint to match so this just makes sure of that. Should we do another approach?

Reply:
But this property isn't used anywhere so why does it matter?